tooling/electron-mksnapshot/src/mksnapshot-run.ts

```diff
@@ -0,0 +1,221 @@
+import fs from 'fs-extra'
+import { StdioOptions, spawnSync } from 'child_process'
+
+import { config } from './config'
+import path from 'path'
+import { track } from 'temp'
+import { processArgsFromFile } from './process-args-from-file'
+
+import debug from 'debug'
+const logInfo = debug('mksnapshot:info')
```

Comment:
Any thoughts about keeping these debugs in a `cypress:mksnapshot:...` (or maybe `tooling:mksnapshot:...`) namespace now that the package is in the monorepo?

Reply:
Good call: [`359c383` (#23284)](https://github.com/cypress-io/cypress/pull/23284/commits/359c383c23e6637a1ffd7edca02a6c2f26596884)